flashinfer/configs/0.2.8/trtllm_fused_moe_NVIDIA_B200.json

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
We need to be careful about these non-python files as we usually forget to package them in the sdist/wheel.
Can we hardcode them in python instead?